workers/ohsome_quality_analyst/api/api.py

```diff
@@ -133,6 +133,7 @@ async def post_indicator(
 async def _fetch_indicator(
     name: str,
     layer_name: str,
+    svg: bool,
```

Comment:
I would name it to something meaningful like: `include_svg`. Also I would suggest using a default value.

For this to work you need to add a API parameter to `get_indicator()` and `post_indicator()` as well. To add this parameter to `post_indicator` you will need to edit `request_models.py`. There a new parameter can be added by creating a new attribute for the `BaseRequestModel` class.

Reply:
Also this will need to be implemented for creating a report as well.

Reply:
Why i need to edit request_models.py? Can't i pass the parameter like in get_indicator? Worked in my tests. Or is there a problem with that?

Reply:
A request to the API for an indicator or report can be done by either sending an HTTP GET request or an HTTP POST request. Your changes have only been done for GET request. 

It is possible to define the query parameter by adding a function parameter to `post_indicator`. But for POST request we are using pydatic data models instead which does some magic in the background. E.g. by adding the `svg` parameter and its type as attribute to the pydatic data model, pydantic will verify the type once a request has been made.  Also FastAPI will use the pydantic data model to generate docs, possible values and examples for the swagger interface.

I hope this makes sense.